Java-Introduction.md

```diff
@@ -0,0 +1,123 @@
+# Introduction
+
+## What is Java?
+
+Java is a programming language developed by Sun Microsystems in 1995, which got later acquired by Oracle. It is now a full platform with lots of standard API's, open source API's, tools, huge developer community and is used to build the most trusted enterprise solutions by big and small companies alike. Android operating system is built fully with Java and its ecosystem. To know more about Java, read [this](https://java.com/en/download/faq/whatis_java.xml) and [this](http://tutorials.jenkov.com/java/what-is-java.html).
+
+## Version
+
+The latest version is [Java 1.8](http://www.oracle.com/technetwork/java/javase/overview/java8-2100321.html) or Java 8, which was released in 2014 with [various improvements](http://www.oracle.com/technetwork/java/javase/8-whats-new-2157071.html) over the previous version, Java 7. This year, Oracle has a target to release Java 9. But for all intents and purposes, we will use Java 8 in this wiki for all tutorials.
+
+Java is also divided into several "Editions" :
+
+* [SE](http://www.oracle.com/technetwork/java/javase/overview/index.html) - Standard Edition - for desktop and standalone server applications
+* [EE](http://www.oracle.com/technetwork/java/javaee/overview/index.html) - Enterprise Edition -  for developing and executing Java components that run embedded in a Java server
+* [ME](http://www.oracle.com/technetwork/java/embedded/javame/overview/index.html) - Micro Edition - for developing and executing Java applications on mobile phones and embedded devices
+
+## Installation
+
+Download the latest Java binaries from the [official website](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html). Here you may face a question, which one to download, JDK or JRE? JRE stands for Java Runtime Environment, which is the platform dependent Java Virtual Machine to run Java codes, and JDK stands for Java Development Kit, which consists of most of the development tools, most importantly the compiler `javac`, and also the JRE. So, for an average user JRE would be sufficient, but since we would be developing with Java, we would download the JDK.
+
+Platform specific installation instructions
+
+### Windows
+
+* Download the relevant .msi file (x86 / i586 for 32bits, x64 for 64bits)
+* Run the .msi file. Its a self extracting executable file which will install Java in your system!
+
+### Linux
+
+* Download the relevant tar.gz file for your system and install :
+
+   ```bash
+   $ tar zxvf jdk-8uversion-linux-x64.tar.gz
+   ```
+* RPM based Linux platforms download the relevant .rpm file and install :
```

Comment:
Link to some site that describes or lists out what RPM is, and how to know if someone's distro is RPM based.